src/buildHTML.js

```diff
@@ -740,6 +755,10 @@ groupTypes.op = function(group, options, prev) {
         large = true;
     }
 
+    // TODO(cbreeden) Currentl mathop atoms have the assumed structure:
+    //   group<atom>.value === { body, symbol, limits, type }
+    //   this may make it difficult for ops that are not either
+    //   a single symbol or list of text.
```

Comment:
It seems like those are all of the cases we handle in `groupTypes.op`... are there other types of operators that we haven't handled yet that you're concerned about?